docs/installation/network-options.md

```diff
@@ -18,39 +18,57 @@ This page describes the following networking options:
+* Using `vxlan` on Rasperry Pi with recent versions of Ubuntu requires [additional preparation](advanced/advanced.md/#raspberry-pi).
+* Using `wireguard-native` as the Flannel backend may require additional modules on some Linux distributions. Please see the [WireGuard Install Guide](https://www.wireguard.com/install/) for details.
+  The WireGuard install steps will ensure the appropriate kernel modules are installed for your operating system.
+  You must ensure that WireGuard kernel modules are available on every node, both servers and agents, before attempting to use the WireGuard Flannel backend.
 
-We recommend that users migrate to the new backend as soon as possible. The migration requires a short period of downtime while nodes come up with the new configuration. You should follow these two steps:
-
-1 - Update the K3s config in all control-plane nodes. The config file `/etc/rancher/k3s/config.yaml` should include `flannel-backend: wireguard-native` instead of `flannel-backend: wireguard`.
-
-2 - Reboot all nodes.
 
   CLI Flag and Value | Description
   -------------------|------------
- `--flannel-backend=vxlan` | (Default) Uses the VXLAN backend. |
- `--flannel-backend=ipsec` | Uses the IPSEC backend which encrypts network traffic. |
- `--flannel-backend=host-gw` |  Uses the host-gw backend. |
- `--flannel-backend=wireguard` | Uses the WireGuard backend which encrypts network traffic. May require additional kernel modules and configuration. (Deprecated and will be removed from version 1.26) |
- `--flannel-backend=wireguard-native` | Uses the WireGuard backend which encrypts network traffic. May require additional kernel modules and configuration. |
- `--flannel-ipv6-masq` | Apply masquerading rules to IPv6 traffic (default for IPv4). Only applies on dual-stack or IPv6-only clusters |
+ `--flannel-ipv6-masq` | Apply masquerading rules to IPv6 traffic (default for IPv4). Only applies on dual-stack or IPv6-only clusters. Compatible with any Flannel backend other than `none`. |
+ `--flannel-external-ip` | Use node external IP addresses as the destination for Flannel traffic, instead of internal IPs. Only applies when --node-external-ip is set on a node. |
+ `--flannel-backend=vxlan` | Use VXLAN to encapsulate the packets. May require additional kernel modules on Raspberry Pi. |
+ `--flannel-backend=host-gw` | Use IP routes to pod subnets via node IPs. Requires direct layer 2 connectivity between all nodes in the cluster. |
+ `--flannel-backend=wireguard-native` | Use WireGuard to encapsulate and encrypt network traffic. May require additional kernel modules. |
+ `--flannel-backend=ipsec` | Use strongSwan IPSec via the `swanctl` binary to encrypt network traffic. (Deprecated; will be removed in v1.27.0) |
+ `--flannel-backend=wireguard` | Use WireGuard via the `wg` binary to encrypt network traffic. May require additional kernel modules and configuration. (Deprecated; will be removed in v1.26.0) |
```

Comment:
I wonder if we should already remove this deprecated backend

Reply:
I'd like to leave it in until it's totally gone